build.rs

```diff
@@ -1,5 +1,13 @@
+use std::env;
+use std::fs;
+use std::path::Path;
+
 extern crate prost_build;
 
 fn main() {
     prost_build::compile_protos(&["src/client.proto"], &["src/"]).unwrap();
+    // Make clippy happy by telling it to ignore any badness in this file.
```

Comment:
🙃

Reply:
gah, figured it out. this file is included in another, so we needed to add the ignore to the `mod` around the `include!`! https://github.com/replit/rfbproxy/pull/28/files#diff-42cb6807ad74b3e201c5a7ca98b911c5fa08380e942be6e4ac5807f8377f87fcR32

Reply:
oh nice!